example/test/get_dom_node_test.dart

```diff
@@ -41,7 +42,7 @@ class SimpleComponent extends react.Component {
     react.div({}, [
       react.span({"ref": "refToSpan"}, "Test"),
       react.span({}, counter),
-      react.button({"onClick": (_) => (getDOMNode() as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
+      react.button({"onClick": (_) => (react_dom.findDOMNode(this) as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
```

Comment:
We could have, and that is actually what has been happening since react-dart 0.9.0, but in that PR it was voted to deprecate it and remove it when ReactJs did to keep better parity, same reason we split react into react and react_dom.